Code/Framework/AzCore/AzCore/Serialization/DataOverlayProviderMsgs.cpp

```diff
@@ -19,8 +19,11 @@ namespace AZ
         nodeStack.push_back(m_dataContainer);
 
         SerializeContext::EnumerateInstanceCallContext callContext(
-            AZStd::bind(&DataOverlayTarget::ElementBegin, this, &nodeStack, AZStd::placeholders::_1, AZStd::placeholders::_2, AZStd::placeholders::_3),
-            AZStd::bind(&DataOverlayTarget::ElementEnd, this, &nodeStack),
+            [this,&nodeStack](void* instance_pointer, const SerializeContext::ClassData* classData, const SerializeContext::ClassElement* classElement)->bool
+            {
+                return ElementBegin(&nodeStack,instance_pointer,classData,classElement);
+            },
+            [this,&nodeStack]()->bool { return ElementEnd(&nodeStack); },
```

Comment:
nitpick: missing space between the lambda capture parameters.
`[this,&nodeStack]`